Dockerfiles/Dockerfile-cmssw

```diff
@@ -1,5 +1,5 @@
 FROM cmssw/el8:x86_64
 
-ADD step2_21034.1_100Events.root /data2/segmentlinking/
+ADD step2_24834.703_100Events.root /data2/segmentlinking/
```

Comment:
just to be sure, how was this file obtained?
24834 workflow has no PU; was it added manually?
It would be more clear to keep the actual step2 matrix number

Reply:
do you usually run just step2 or remake step1 as well?
running the matrix with `-i all` will instruct to use an existing GEN-SIM and can save a bit of time (with the only complication of its availability at the local site or via xrootd)

Reply:
Thank you, Slava.

I switched to using 29834.1 plus LST. I had to go down to only 75 events because the file is larger now, but it seems to be working now.

Reply:
I think we can trim the file to get rid of products that we do not need; but 75 should be OK for now to get going.
Do you have a large file somewhere on uaf/ucsd? I could trim it down manually.

Reply:
I used a 4-event test of 29834.1 (something that doesn't take long).
The step2.root original is 450MB

After the following update in the step2 command: 
- change `--pileup AVE_200_BX_25ns,'{"B":(-2,0)}'`
- add `--outputCommands='drop SimClusters_*_*_*','drop MtdSim*_*_*_*','drop *_TT*_*_*','drop *_*HGCal*_*_*','drop EcalTime*_*_*_*','drop *_hlt*_*_*','drop *_l1t*_*_*','drop *_simEcal*_*_*'`

I get the output file of 151 MB.

It's probably pretty safe still to use `{"B":(-1,0)}`  I vaguely recall that only around a % of tracker hits come from earlier bunch crossings (-2 or earlier) and -1 has gives around 10%.

Reply:
Ah okay, I'll give that a try. Thank you!